AnnoDesigner/Helper/RoadSearchHelper.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Windows;
+using AnnoDesigner.Core.Models;
+
+namespace AnnoDesigner.Helper
+{
+    public static class RoadSearchHelper
+    {
+        private static void DoNothing(AnnoObject objectInRange) { }
+
+        public static Dictionary<double, Dictionary<double, AnnoObject>> PrepareGridDictionary(IEnumerable<AnnoObject> placedObjects)
+        {
+            var dictionary = new Dictionary<double, Dictionary<double, AnnoObject>>();
```

Comment:
I think it is better to name the variables according to their function. So maybe `var result =` would be better.
What do you think?

Reply:
Sure, it carries very little information.